File: LeetCode_scripts/python/two-sum/two_sum.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        position = {}

        for i, num in enumerate(nums):
            if (target - nums[i]) in position.keys():
                result = [position[target - nums[i]], i]
                return result

            position[num] = i
        
        return []

File: LeetCode_scripts/go/two-sum/twoSum.go
package twosum

func twoSum(nums []int, target int) []int {
	indices := []int{}

	for i := 0; i < len(nums); i++ {
		for j := range nums {
			if i == j {
				continue
			} else if nums[i]+nums[j] == target {
				indices = append(indices, i, j)
				return indices

			}

		}
	}
	return nil

}
